.github/workflows/release.yml

```diff
@@ -51,6 +49,9 @@ jobs:
           - goos: android
             goarch: arm64
             asset_arch: arm64
+          - goos: android
+            goarch: arm
```

Comment:
this one also requires to say which arm version.
in this case: `goarm: 7`

Reply:
done